src/borg/archiver.py

```diff
@@ -306,8 +306,11 @@ def do_init(self, args, repository):
     def do_check(self, args, repository):
         """Check repository consistency"""
         if args.repair:
-            msg = ("'check --repair' is an experimental feature that might result in data loss." +
-                   "\n" +
+            msg = ("This is a potentially dangerous functionality.\n"
```

Comment:
I'd say `This is a potentially dangerous function.`
```suggestion
            msg = ("This is a potentially dangerous function.\n"
```

Reply:
fixed